internal/clients/http/azure/azure_client.go

```diff
@@ -10,61 +10,55 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armsubscriptions"
 	"github.com/RHEnVision/provisioning-backend/internal/clients"
 	"github.com/RHEnVision/provisioning-backend/internal/config"
-	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
-	"github.com/rs/zerolog"
 )
 
-type azureClient struct {
+type client struct {
 	subscriptionID string
 	credential     *azidentity.ClientSecretCredential
 }
 
 func init() {
-	clients.GetAzureClient = newAzureClient
+	clients.GetAzureClient = newCustomerClient
```

Comment:
In GCP(and aws) the customer function is named newGCPClient
(Without a change) can we be consistent?

Reply:
to add my 2c I like `newAzureClient` more, as Customer might be missleading, if this service were to be used outside of it's the RH context

Reply:
@lzap Can we keep newAzureClient?

Reply:
Absolutely, adding an extra commit.